File: SNS.Web/src/app/jobs/company-administrators/contracts/company-administrator.dto.ts
import { ProfileSnapshotDto } from '../../../profiles/profiles/contracts/profile-snapshot.dto';
import { CompanyRole } from '../../enums/company-role.enum';

export interface CompanyAdministratorDto {
    id: string;
    companyId: string;
    profileId: string;
    profile: ProfileSnapshotDto;
    adminRole: CompanyRole;
}

File: backend/SNS.Application/Jobs/CompanyAdministrators/Contracts/CompanyAdministratorDto.cs
using SNS.Application.Profiles.Profiles.Contracts;
using SNS.Domain.Jobs.Enums;

namespace SNS.Application.Jobs.CompanyAdministrators.Contracts;

public sealed record CompanyAdministratorDto(
    Guid Id,
    Guid CompanyId,
    Guid ProfileId,
    ProfileSnapshotDto Profile,
    CompanyRole AdminRole
);
